linux_os/guide/system/software/system-tools/package_rng-tools_installed/rule.yml

```diff
@@ -18,6 +18,9 @@ identifiers:
     cce@rhel7: 82969-7
     cce@rhel8: 82968-9
 
+references:
+    srg: SRG-OS-000480-GPOS-00227
```

Comment:
Really torn about this one. rng-tools will help make random number generation faster and could improve the *availability* of systems performing such tasks (e.g. generating PKI certificates).

But should this helper tool be mandated across DoD? Leaning towards no.

@tedbrunell could you add some reasoning to the rule selection?

Reply:
Besides to obvious answer of "It's in the OSPP profile, therefore it should map to something in the SRG/STIG" I think that this package creates better entropy for functions that need it. https://access.redhat.com/articles/1314933

I think the requirement should stay.

Reply:
I think that if you study the state of entropy on any linux system, you will realize how bad things are. This daemon helps the situation by providing a jitter entropy source which has been studied extensively. Try this:

cat /dev/random

Try it without the daemon running and then with.

Reply:
Agreed - lets keep it. Thanks for the discussion!